kroxylicious-operator/src/main/resources/META-INF/fabric8/virtualkafkaclusters.kroxylicious.io-v1.yml

```
@@ -143,4 +143,24 @@ spec:
                       - observedGeneration
                       - reason
                       - message
-                      - type
\ No newline at end of file
+                      - type
+                ingresses:
+                  type: array
+                  description: -|
+                    Ingresses is a list of ingresses that are associated with the virtual cluster. It reports
+                    the name of each ingress associated with the virtual cluster and the boostrap that
+                    a Kafka client will use to connect to the virtual cluster.
+                  default: []
+                  items:
+                    type: object
+                    required: [ "name" ]
+                    properties:
+                      name:
+                        description: Name is the name of the ingress.
+                        type: string
+                      bootstrap:
+                        description: -|
+                          Bootstrap that a Kafka client would use to connect to the virtual cluster. The boostrap
+                          field will be absent if the address is not yet available. This may mean that work by
+                          another operator to assign the address is still pending.
```

* Bootstrap is not really a noun with a well-known meaning. It's an adjective which modifies what this _actually_ is, which is an address. Maybe we should rename the property itself?
* Be explicit that this is only used for the _initial_ connection.
* "This may mean that work by another operator to assign the address is still pending." doesn't really help the user at all (we can assume they have no idea about operators, or whether that "other operator" is part of the kroxy operator or something else). All that really matters is that this property might be missing for a time.

```suggestion
                        description: -|
                          The address that a Kafka client would use make its initial connection to the virtual cluster. 
                          This property will be absent if the address is not yet available.
```